### DISCUSSION TITLE
phpspreadsheet errors thrown by JED Checker

### DISCUSSION THREAD

**Discussion by alan123489** [joomengine/Joomla-Component-Builder#700] [Q&A] [answered]
I've been trying to get a component approved by JED.

It throw over 300 errors base on phpspreadsheet:

com_mediadistributor/libraries/phpspreadsheet/vendor/markbaker/complex/classes/src/functions/acos.php
The JEXEC security check was not found in this file.

and

 /com_mediadistributor/libraries/phpspreadsheet/vendor/autoload.php
GPL or compatible license was not found

these are repeated over and over.  Since phpspreadsheet is an external program is there anything that  can be done about this and if not can it be removed from the compile in some way?

thanks

Alan

**Comment by Llewellynvdm** [comment, marked as answer]
The JED allows this with 3rd party libraries being used in your component.

**Reply by alan123489** [reply]
thanks - I'll try the submission and see if there is any hold up.
